### PR TITLE
add related arm64 syscall for vfs2

### PR DESCRIPTION
**Pull request by zhlhahaha**
arm64 vfs2: Add support for io_submit/fallocate/
sendfile/newfstatat/readahead/fadvise64

Signed-off-by: Howard Zhang <howard.zhang@arm.com>

### CODE REVIEW

**Comment by zhlhahaha** [comment]
CC @lubinszARM

**Comment by zhlhahaha** [comment]
@avagin Hi Andrei would you mind review it when you have time? thanks

File: pkg/sentry/syscalls/linux/vfs2/vfs2.go [hunk 1]
```diff
@@ -163,6 +163,7 @@ func Override() {
 
 	// Override ARM64.
 	s = linux.ARM64
+	s.Table[2] = syscalls.Supported("io_submit", IoSubmit)
```
Comment (avagin) [inline pkg/sentry/syscalls/linux/vfs2/vfs2.go:166]: should it be PartiallySupported?

Comment (zhlhahaha) [reply pkg/sentry/syscalls/linux/vfs2/vfs2.go:166]: > should it be PartiallySupported?

Yes, right, fadvise64 and fallocate should be also PartiallySupported, I will fix it.

**Comment by lubinszARM** [comment]
Related issue: https://github.com/google/gvisor/issues/2945

**Pull request merged**